src/main/java/org/elasticsearch/action/search/SearchScrollRequest.java

```diff
@@ -39,6 +42,9 @@
     private String scrollId;
     private Scroll scroll;
```

Comment:
Can you make the setters to the `scrollId` and `scroll` fields package protected? Now we have SearchScrollRequestBuilder and that is what should be used to set the options for a scroll search request.